tools/ci/run_unit_tests.sh

```diff
@@ -19,4 +19,5 @@ sudo chown -R $USER test-results
 # circleCI can be particularly slow sometimes since some time around 2021-06
 export OPENQA_TEST_TIMEOUT_SCALE_CI=3
 export EXTRA_PROVE_ARGS="-v"
+export OPENQA_FULLSTACK_TEMP_DIR=test-results/fullstack
```

Comment:
That needs to be an absolute path:
```suggestion
export OPENQA_FULLSTACK_TEMP_DIR=$PWD/test-results/fullstack
```